docs/arcadia-users-group/20240305-github-templates/lesson.md

```diff
@@ -0,0 +1,88 @@
+# Introduction to GitHub Templates
+
+GitHub templates provide a structured way to create new repositories, ensuring consistency, efficiency, and adherence to best practices across projects.
+GitHub templates are predefined repository setups that can be used to start new projects with a consistent set of files, folders, and configurations.
+This not only saves time but helps make sure that all projects follow Arcadia's coding and documentation standards.
+Templates typically include everything from README files, installation guides, contribution guidelines, and more.
+
+In this lesson, we will focus on Arcadia's [`python-analysis-template`](https://github.com/Arcadia-Science/python-analysis-template).
+Arcadia also has an [`r-analysis-template`](https://github.com/Arcadia-Science/r-analysis-template), a [`snakemake-template`](https://github.com/Arcadia-Science/snakemake-template), and a [`nextflow-template`](https://github.com/Arcadia-Science/nextflow-template).
+
+## Creating a new repository using a template
+
+### Choosing the Right Template
+
+Depending on the analysis type or the technology stack used in your project, select the appropriate template. 
+For instance, use `r-analysis-template` for R-based analyses, `python-analysis-template` for Python projects, `snakemake-template` for workflows managed with Snakemake, and `nextflow-template` for Nextflow pipelines.
+
+<details>
+<summary>Why isn't there a Python package template?</summary>
+
+We could create one. 
+We haven't generated one yet because there are a ton templates out there that could be used and we haven't created many Python packages yet as an organization.
+If you find yourself in need of a Python template, try out <a href=https://github.com/TezRomacH/python-package-template>an existing template</a> and make note of what works well and what doesn't.
+We can then use that to generate a future organization standard template.
+
+</details>
+
+### Repository Initialization
+
+There are two ways to create a new repo with a template.
+First, you can navigate to the template you want to start from (e.x. https://github.com/Arcadia-Science/python-analysis-template) and click the green "Use this template" button in the upper right hand corner.
+This will take you to the New Repository launch screen with the "Repository template" section already filled out.
+Alternatively, you can the "+" button in the upper right hand corner of any GitHub page and select "New repository" from the drop down menu.
```

Comment:
```suggestion
Alternatively, you can click the "+" button in the upper right hand corner of any GitHub page and select "New repository" from the drop down menu.
```
I think there is a word missing here. I'm not sure `click` is better than `use` or `select`, or something different though.